lib/addons/ack/index.ts

```diff
@@ -0,0 +1,108 @@
+// lib/certmanager_addon.ts
+import { ManagedPolicy } from 'aws-cdk-lib/aws-iam';
+import { Construct } from 'constructs';
+import merge from "ts-deepmerge";
+import { ClusterInfo, Values } from "../../spi";
+import { createNamespace, setPath } from "../../utils";
+import { HelmAddOn, HelmAddOnProps, HelmAddOnUserProps } from "../helm-addon";
+/**
+ * User provided option for the Helm Chart
+ */
+export interface AckAddOnProps extends HelmAddOnUserProps {

```

Comment:
We need to design it a bit better. I suggest just letting customers pass the service name, like iam or rds as one field and we apply the logic to inject it to the repo, chart, release, sa. 

We should also supply an enum of supported service names:

```typescript
export enum AckServiceName {
   IAM = "iam",
   RDS = "rds" // etc.
}
```

Another thing that would be really helpful is managed policy mapping with each service, e.g. IAM -> IAMFullAccess

Reply:
This will make it very difficult to maintain it as ACK constantly adds one service after the other. Currently 47 services are supported today. Adding and maintaining is each month is something we dont want to do. Lets talk about it.